risk-map/yaml/controls.yaml

```
@@ -80,6 +80,23 @@ controls:
     risks:
       - DP
       - UTD
+    mappings:
+      mitre-atlas:
+        - AML.M0013
+      nist-ai-rmf:
+        - MS-2.7
+        - MS-2.8
+    lifecycleStage:
+      - data-preparation
+      - model-training
+      - evaluation
+    impactType:
+      - integrity
+      - reliability
```

Feels like the impacts should include `privacy` and `fairness`

```suggestion
    impactType:
      - integrity
      - reliability
      - safety
      - privacy
      - fairness
```